### PR TITLE
Fix standard longitude for Lambert projection & add Polar Stereographic and Mercator

### LINKED ISSUE #116: Standard longitude bug
**Describe the bug**
I have lambert grid where the standard longitude is -10, whereas the center_point longitude = 2. In GIS4WRF, I only have the possibility to set the center_longitude. Inspection of namelist.wps learned me that the standard longitude is set to 2 as well. This is incorrect. 

Changing the standard longitude in namelist.wps, and then "import from namelist" did not show the correct grid, aligned with the -10 longitude line. 

**Expected behavior**
Extra box to provide the standard longitude. In the above example, the grid should be aligned (by rotation) with the longitude line -10, and the center_longitude of the grid should be equal to 2.

### PR DESCRIPTION
**Pull request by letmaik**
This fixes #116 and also adds the two missing WRF projections.

**Note:** The gis4wrf [tutorial](https://gis4wrf.github.io/tutorials/) has to be changed as well because standard longitude is now required for Lambert.

### CODE REVIEW

File: gis4wrf/core/transforms/wps_namelist_to_project.py [hunk 1]
```diff
@@ -48,13 +48,20 @@ def convert_nml_to_project_domains(nml: dict) -> List[dict]:
...
     if map_proj == 'lat-lon':
-        if standlon != 0.0:
+        if stand_lon != 0.0:
             raise UnsupportedError('Rotated lat-lon projection is not supported')
         crs = CRS.create_lonlat()
     elif map_proj == 'lambert':
-        # It doesn't matter what the origin is. See wps_binary_to_gdal.py for details.
-        origin = LonLat(lon=standlon, lat=(truelat1 + truelat2)/2)
+        # It doesn't matter what the latitude origin is.
+        # Longitude is standard longitude and matters.
```
Comment (dmey) [inline gis4wrf/core/transforms/wps_namelist_to_project.py]: ? Rephrase.

File: gis4wrf/metadata.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [general]
 name=GIS4WRF
-version=0.13.2
+version=0.14.0
```
Comment (dmey) [inline gis4wrf/metadata.txt]: I would not bump the version here. I would do that in a different PR when 14.0 is ready and all the changes have been added.
```suggestion
version=0.13.2
```

**Comment by dmey** [comment]
Docs and tutorial updated by https://github.com/GIS4WRF/gis4wrf.github.io/commit/d49fb3c27377dfb81265747c2f631e05ee4b279d to reflect this change.

File: gis4wrf/core/transforms/wps_namelist_to_project.py (hunk 1 above)
Comment (letmaik) [reply gis4wrf/core/transforms/wps_namelist_to_project.py]: I re-wrote the existing text in wps_binary_to_gdal.py and then referred to that from here. It should hopefully now be clear for future reference.

Comment (dmey) [reply gis4wrf/core/transforms/wps_namelist_to_project.py]: Agree. Also the use of `arbitrary_longitude_origin = 0`  is more informative!

**Comment by letmaik** [comment]
Once positive feedback comes in from Hugo in #116 this can be (squash) merged.

**Comment by letmaik** [comment]
[Confirmation](https://github.com/GIS4WRF/gis4wrf/issues/116#issuecomment-471452112) received.
